### ISSUE TITLE
License requirement

### ISSUE THREAD

**Issue by 0fflineDocs** [MicrosoftDocs/microsoft-365-docs#11319]
Hello,

On this page it says "Applies to Microsoft Defender for Endpoint Plan 1"

But looking here, it's clearly stated that Defender for Endpoint Plan 2 is required.
https://learn.microsoft.com/en-gb/microsoft-365/security/defender-endpoint/microsoft-cloud-app-security-integration?view=o365-worldwide

Both pages should reflect the correct licensing requirements.

Thanks in advance!

---
#### Document details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: ef17017b-6f55-d9cf-1560-1bc7bec027ab
* Version Independent ID: 6d2f7adc-db74-b3df-859c-e27a142332ee
* Content: [Microsoft Defender for Cloud Apps integration overview](https://learn.microsoft.com/en-gb/microsoft-365/security/defender-endpoint/microsoft-cloud-app-security-integration?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender-endpoint/microsoft-cloud-app-security-integration.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/microsoft-cloud-app-security-integration.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @mjcaparas
* Microsoft Alias: **macapara**

**Comment by msbemba** [comment]
@0fflineDocs Thank you for your feedback. Looks like both documents are the same. Could you please provide the alternate link to verify and update?

Thanks
Sri

**Comment by 0fflineDocs** [comment]
You are correct, this is the one: https://learn.microsoft.com/en-us/defender-cloud-apps/mde-integration#prerequisites


MDE P1 and P2 is stated here: https://learn.microsoft.com/en-gb/microsoft-365/security/defender-endpoint/microsoft-cloud-app-security-integration?view=o365-worldwide

But here clearly P2 (or business) is a requirement: 
https://learn.microsoft.com/en-us/defender-cloud-apps/mde-integration#prerequisites

Thanks in advance.

**Comment by msbemba** [comment]
@0fflineDocs Thank you for your feedback. We have made changes to the document via PR#https://github.com/MicrosoftDocs/microsoft-365-docs/pull/11347

Once the author approves, the changes will be live.

Thanks
Sri

**Comment by yogkumgit** [comment]
@msbemba Thank you very much for the contribution and updating the documentation with PR. @0fflineDocs Hope this update is helpful for you. Thanks for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.
